orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/pipeline/WebhookStage.java

```diff
@@ -84,7 +85,7 @@ public void onFailureStages(@Nonnull StageExecution stage, @Nonnull StageGraphBu
     public String url;
     public Object payload;
     public Map<String, Object> customHeaders;
-    public List<Integer> failFastStatusCodes;
+    public List<Integer> failFastStatusCodes = List.of(504);
```

Comment:
default to having a gateway timeout result in a fail fast by default

Reply:
Perhaps https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.html#GATEWAY_TIMEOUT instead to make that more obvious?